### ISSUE TITLE
WCAG 2.0

### ISSUE THREAD

**Issue by banukutlu** [psu-libraries/psulib_blacklight#26]
In GitLab by @rkt6 on Apr 4, 2018, 12:08

Starting off this ticket as a place to track/record the regular accessibility testing conducted through I-Tech's processes. This serves as a place to log issues discovered and manage fixing them.

**Comment by ruthtillman** [comment]
Useful links:

Mike Tribone's open tickets on Blacklight (to do with Accessibility). We can check on them and whether they've been resolved before starting this milestone: https://github.com/projectblacklight/blacklight/issues/created_by/mtribone

May 2018 and Jan 2019 accessibility reports:
https://psu.box.com/s/cn2xb0agw8oebun4zmxxg4dnq4pyp3n9
https://psu.box.com/s/v6xagyvmrxuwu1h9zw4kc8y6k2kcp5wn

**Comment by banukutlu** [comment]
closing this since we added #304 #303 #302, we will add more if we have further issues after the next accessibility testing
